roles/storage-cinder/tasks/deprovision.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Delete Service Account
+  command: kubectl delete serviceaccount {{service_account}} -n {{namespace}}
+  when: sa.stdout == ""
+
+- name: Revoke anyuid access to {{service_account}} SA if using OCP
+  command: oc adm policy remove-scc-to-user anyuid -z {{service_account}} -n {{namespace}}
+  when: cluster == "openshift"
+
+- name: Delete {{privileged_service_account}} serviceaccount
+  command: kubectl create serviceaccount {{privileged_service_account}} -n {{ namespace }}
+  when: psa.stdout == ""
+
+- name: Revoke privileged access to {{privileged_service_account}} serviceaccount
+  command: oc adm policy remove-scc-to-user privileged system:serviceaccount:{{ namespace }}:{{privileged_service_account}}
+  when: cluster=="openshift"
+
+- name: Select a host for Mariadb hostpath
+  command: kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="Hostname")].address}'
+  register: node_hostname
+
+- name: Label node to host Mariadb
```

Comment:
You want to remove the label right?  Also, can't you get all the nodes that have the mariadb label as a query?  Here you assume that the get nodes verb always returns them in the same order and that should not be relied upon.